docs/conf.py

```diff
@@ -0,0 +1,323 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
```

Comment:
Not agreed. This is how we do it for the Invenio modules, or at least, how I got it described when designing the cookiecutter. If we change it here, someone also might to change the cookiecutter.